releases/release-1.19/README.md

```diff
@@ -1,3 +1,13 @@
+---
+title: "Kubernetes 1.19 Release Information"
+weight: 98
+slug: "release"
+aliases: [ "/release" ]
```

Comment:
Just an FYI - The canonical link on the contributor site will be `/resources/release` but adding this alias means `/release` from the root will auto-direct to the page. The eventual URL will be something like this:
https://k8s.dev/release